### ISSUE TITLE
How can I ignore multiple labels during training?

### ISSUE THREAD

**Issue by yuanyuangoo** [open-mmlab/mmsegmentation#2094]
What should be changed in the config file to ignore more than 2 labels during training?

Where should I put avg_non_ignore and ignore_index?

**Comment by xiexinch** [comment]
Hi @yuanyuangoo,
Currently, does not support ignoring more than two labels.

`avg_non_ignore` should be set at the loss configuration, and `ignore_index` should be set at the decode_head configuration.

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/losses/cross_entropy_loss.py#L225

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/decode_heads/decode_head.py#L49

**Comment by yuanyuangoo** [comment]
Hi, May I ask how to exclude a label in evaluation?

**Comment by xiexinch** [comment]
Try to set the `ignore_index` to your dataset configuration.

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L89

For example, set the `ignore_index` parameter  to the val dataset: 

https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/datasets/cityscapes.py#L43

**Comment by yuanyuangoo** [comment]
Thanks.

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
